client/my-sites/site-settings/media-settings/carousel-settings.jsx

```diff
@@ -0,0 +1,43 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+/**
+ * Internal dependencies
+ */
+import FormSelect from 'components/forms/form-select';
+import FormLabel from 'components/forms/form-label';
+import FormCheckbox from 'components/forms/form-checkbox';
+
+const CarouselSettings = props => {
+	return (
+		<div className="media-settings__module-settings is-indented">
+			<FormLabel>
+				<FormCheckbox
+					name="carousel_display_exif"
+					checked={ props.carousel_display_exif || false }
+					onChange={ props.handleCheckbox }
+					disabled={ props.submittingForm } />
+				<span>{ props.translate( 'Show photo metadata (Exif) in carousel, when available' ) }</span>
```

Comment:
Alternate string suggestion: 
* `translate( 'Show photo metadata (<a href="http://en.wikipedia.org/wiki/Exchangeable_image_file_format" target="_blank">Exif</a>) in carousel, when available.' )`&mdash; translations: **28**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/jetpack-site-settings-ui)